packages/eslint-plugin-pf-codemods/src/rules/v6/noDuplicateImportSpecifiers/no-duplicate-import-specifiers.test.ts

```diff
@@ -0,0 +1,64 @@
+const ruleTester = require("../../ruletester");
+import * as rule from "./no-duplicate-import-specifiers";
+
+ruleTester.run("no-duplicate-import-specifiers", rule, {
+  valid: [
+    {
+      // we care only about imports from "@patternfly/react-core"
+      code: `import { Button, Button } from "somewhere"`,
```

Comment:
And maybe a valid test where Component is imported from react-core, but also react-core/[deprecated|next]?